### ISSUE TITLE
log-prior in README

### ISSUE THREAD

**Issue by cscherrer** [AnderGray/TransitionalMCMC.jl#8]
Hi, the README has 

```julia
priorDen(x) = logpdf(Uniform(lb, ub), x[1,:]) .* logpdf(Uniform(lb, ub), x[2,:])
```

A couple of things here:
- I think the `.*` should now be `.+`
- What do you think of `logprior` instead of `priorDen`? That could make it a little clearer IMO

**Comment by AnderGray** [comment]
Yikes good spot. Yes to both. Similar changes for the examples.

Thanks!

**Comment by AnderGray** [comment]
Closed with [PR#9](https://github.com/AnderGray/TransitionalMCMC.jl/pull/9)
